packages/app-desktop/gui/ConfigScreen/controls/plugins/PluginBox.tsx

```diff
@@ -137,9 +137,10 @@ export default function(props: Props) {
 	const item = props.item ? props.item : manifestToItem(props.manifest);
 
 	const onNameClick = useCallback(() => {
-		if (!props.item.manifest.homepage_url) return;
-		bridge().openExternal(props.item.manifest.homepage_url);
-	}, [props.item]);
+		const manifest = item.manifest;
+		if (!manifest.homepage_url) return;
+		bridge().openExternal(manifest.homepage_url);
+	}, [props.item, props.manifest]);
```

Comment:
Also please update the dependencies. I believe only `item` is now needed as a dependency.

Reply:
Also done